### ISSUE TITLE
Problems on the Windows machine

### ISSUE THREAD

**Issue by nescha** [felixmosh/i18next-hmr#12]
Scenario: Same App code with almost the same configurations on macOS, Ubuntu and Windows (configuration my differ in minor module versions). Execution happens in the same Chrome browser.
Doesn't work on Windows - after translation change in the locale JSON file and after a page refresh, changes are not applied.

The one noticeable difference is that console.log about the file change does not trigger in Windows unlike in other environments:
Windows
```
[Fast Refresh] rebuilding
[Fast Refresh] done
```

vs. e.g. Ubuntu

```
[Fast Refresh] rebuilding
[I18NextHMR] Got an update with 'rs', 'common'
[I18NextHMR] Update applied successfully
[Fast Refresh] done
```

I tried with changing backslash vs. forward slash in paths in next.config.js (localesDir) and i18n.js (localePath), but with no success.

Any hints?

**Comment by felixmosh** [comment]
Thank you for using this lib 🙏🏼

I didn't tested this lib on windows (I will able to later today).
I will investigate it later today and ping if I need more info...

My guess is that it is related to what you suspected (back slashes)

**Comment by felixmosh** [comment]
As you've suggested lang / namespace derivation function wasn't supported back slashes. 
It fixed in 1.5.5. 
Thank you for reporting this issue 🙏🏼

**Comment by nescha** [comment]
Great! Works now!
Thank you!
